1_fetch/src/fetch_model_RMSEs.R

```diff
@@ -2,10 +2,8 @@
 
 library(sbtools)
 
-fetch_data <- function(out_dir, out_fname, sb_id, sb_files) {
-  # Construct filepath
-  dir.create(out_dir)
-  out_full_file_path <- file.path(out_dir, out_fname)
+fetch_and_load_data <- function(out_fpath, sb_id, sb_files) {
   # Retrieve data
-  item_file_download(sb_id, names = sb_files, destinations = out_full_file_path, overwrite_file = TRUE)
+  item_file_download(sb_id, names = sb_files, destinations = out_fpath, overwrite_file = TRUE)
+  raw_data <- readr::read_csv(out_fpath, col_types = 'iccd')
```

Comment:
Since these are two distinct steps - download and then read - I would actually suggest keeping them as different targets, so not combined into one function here. I think what you had in `process_data()` where you read in the site file works, or you could have two targets: one that calls `fetch_data()` and one that calls `load_data()`.